modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -248,6 +248,31 @@ PolygonConcentricCircleMeshGeneratorBase::PolygonConcentricCircleMeshGeneratorBa
   declareMeshProperty<std::vector<Real>>("azimuthal_angle_meta", std::vector<Real>());
   declareMeshProperty<Real>("max_radius_meta", 0.0);
 
+  const unsigned short tri_order = _tri_elem_type == TRI_ELEM_TYPE::TRI3 ? 1 : 2;
+  const unsigned short quad_order = _quad_elem_type == QUAD_ELEM_TYPE::QUAD4 ? 1 : 2;
+  // 1. If the central elements are quad, then no triangular elements are generated;
+  // 2. If the generated mesh has only one radial layer of triangular elements, then no
+  // quad elements are generated; (For PCCMG, that layer must be background)
+  // 3. Otherwise, both types of elements are generated.
+  _order = quad_order;
+  if (_quad_center_elements)
+  {
+    if (tri_order != quad_order)
+      _tri_elem_type = quad_order == 1 ? TRI_ELEM_TYPE::TRI3 : TRI_ELEM_TYPE::TRI7;
```

Comment:
I'd personally default to TRI6 rather than TRI7, but TRI7 is fine if you prefer.

Reply:
(My reasoning being that Tri6 and Quad9 are the simple, "canonical" P^p and Q^p elements; whereas with isoparametric Lagrange Tri7 is "P^p plus a weird bubble function", and Quad8 is "P^p plus most of Q^p but not all")

Reply:
Makes sense. Will do.